marlowe-test/src/Spec/Marlowe/Semantics/Arbitrary.hs

```diff
@@ -387,18 +386,15 @@ randomRoleNames =
   , "Nonso Ernie Blanka"
   , "Umukoro Alexander Columb"
   , "Urbanus Roland Alison Ty Ryoichi"
-  , "Alcippe Alende Blanka Roland Dafne"  -- NB: Too long for ledger.
```

Comment:
Removed: `Arbitrary` instances should not produce invalid values.

Reply:
We do need negative tests though. Do we still have those? How do we go about them?

Reply:
I don't think we explicitly had them in the first place - because the arbitrary instances sometimes gave invalid data. That is sometimes what you want, sometimes not. For testing handling of invalid data, we should explicitly test those cases and use custom `Gen`s that produce invalid data on purpose.